### DISCUSSION TITLE
Count compressor starts

### DISCUSSION THREAD

**Discussion by nicx** [raomin/ESPAltherma#329] [Q&A]
Is it possible to count the compressor starts? Maybe is there already a value to get this info?
Thanks!

**Comment by nicx** [comment]
any hint?

**Reply by KalaNagHTD** [reply]
Just count the events where the compressor frequency changes from zero to any positive value.

**Reply by hilmar65** [reply]
Hello Karl,
what’s the syntax of your trigger?
Is ist so correct?

platform: state
entity_id:
  - sensor.reserve_channel_1_power
from: "0"
to: ">0"
for:
  hours: 0
  minutes: 5
  seconds: 0

Thank you for your answer.

Greets

Hilmar

**Reply by KalaNagHTD** [reply]
That‘s my JS Blockly in ioBroker

![IMG_5232](https://github.com/raomin/ESPAltherma/assets/77481230/433a7242-d69b-4e54-a2c4-70a62a3b3f34)

**Reply by hilmar65** [reply]
Thank you Karl,
I'll try, to "translate" it in Home Assistant.

Greets

Hilmar

**Reply by MichaelST01** [reply]
@KalaNagHTD Which value is the „Verdichter“ Object?

**Reply by KalaNagHTD** [reply]
„Verdichter“ is „INV Frequency (rps)“ {0x30,0,152,1,-1}

**Reply by tomaszsluszniak** [reply]
What about the `{0x10,1,305,1,-1,"Startup Control"}` parameter? Isn’t that the parameter which tells that the compressor starts?
